sway-lib-std/src/array_conversions/u64.sw

```diff
@@ -207,21 +235,25 @@ impl u64 {
         let g = bytes[6];
         let h = bytes[7];
 
-        asm(a: a, b: b, c: c, d: d, e: e, f: f, g: g, h: h, i: 0x8, j: 0x10, k: 0x18, l: 0x20, m: 0x28, n: 0x30, o: 0x38, r1, r2, r3) {
-            sll  r1 a o;
-            sll  r2 b n;
-            or   r3 r1 r2;
-            sll  r1 c m;
-            or   r2 r3 r1;
-            sll  r3 d l;
-            or   r1 r2 r3;
-            sll  r2 e k;
-            or   r3 r1 r2;
-            sll  r1 f j;
-            or   r2 r3 r1;
-            sll  r3 g i;
-            or   r1 r2 r3;
-            or   r2 r1 h;
+        asm(
```

Comment:
this looks a little odd... having them every argument on their own line also seems excessive..? What does rust-fmt do in this situation?